packages/nouns-webapp/src/components/VoteCard/index.tsx

```diff
@@ -19,33 +20,73 @@ interface VoteCardProps {
   percentage: number;
   nounIds: Array<string>;
   variant: VoteCardVariant;
+  delegateView: boolean;
+  delegateGroupedVoteData:
+    | { delegate: string; supportDetailed: 0 | 1 | 2; nounsRepresented: string[] }[]
+    | undefined;
 }
 
 const VoteCard: React.FC<VoteCardProps> = props => {
-  const { proposal, percentage, nounIds, variant } = props;
-  const isMobile = isMobileScreen();
+  const { proposal, percentage, nounIds, variant, delegateView, delegateGroupedVoteData } = props;
+  const isMobile = window.innerWidth < 1200;
 
   let titleClass;
   let titleCopy;
   let voteCount;
+  let supportDetailedValue: 0 | 1 | 2;
   switch (variant) {
     case VoteCardVariant.FOR:
       titleClass = classes.for;
       titleCopy = <Trans>For</Trans>;
       voteCount = proposal.forCount;
+      supportDetailedValue = 1;
       break;
     case VoteCardVariant.AGAINST:
       titleClass = classes.against;
       titleCopy = <Trans>Against</Trans>;
       voteCount = proposal.againstCount;
+      supportDetailedValue = 0;
       break;
     default:
       titleClass = classes.abstain;
       titleCopy = <Trans>Abstain</Trans>;
       voteCount = proposal.abstainCount;
+      supportDetailedValue = 2;
       break;
   }
 
+  const { library } = useEthers();
+  const [ensCached, setEnsCached] = useState(false);
+
+  // Pre-fetch ENS  of delegates (with 30min TTL)
+  // This makes hover cards load more smoothly
+  useEffect(() => {
+    if (!delegateGroupedVoteData || !library || ensCached) {
+      return;
+    }
+
+    delegateGroupedVoteData.forEach((delegateInfo: { delegate: string }) => {
```

Comment:
Could be improved by only fetching ENS info for delegates that do not exist in local storage.